test/regress_dns.c

```diff
@@ -1012,12 +1012,11 @@ static struct event_base *be_connect_hostname_base = NULL;
 static void
 be_getaddrinfo_server_cb(struct evdns_server_request *req, void *data)
 {
-	int i;
 	int *n_got_p=data;
 	int added_any=0;
 	++*n_got_p;
 
-	for (i=0;i<req->nquestions;++i) {
+	for (int i = 0; i < req->nquestions; ++i) {
```

Comment:
autotools has `-Wdeclaration-after-statement`, so this will report a warning (looks like you tested it only under `cmake` since it doesn't have this requirement)

Reply:
can you avoid mixing them for now?

Reply:
> can you avoid mixing them for now?

Done.